src/Sulu/Bundle/MediaBundle/Entity/CollectionRepository.php

```diff
@@ -295,12 +293,10 @@ private function getIdsQuery(
     ) {
         $queryBuilder = $this->createQueryBuilder('collection')
             ->select($select)
-            ->leftJoin('collection.children', 'collectionChildren')
-            ->where('collection.depth <= :depth1 OR collectionChildren.depth <= :depth2');
```

Comment:
Maybe we should keep the children, to be able to select a specific collection on the "move-media"-overlay. However, when loading the children we get problems with the limit.